store/sqlstore/user_store.go

```diff
@@ -319,23 +320,36 @@ func (us SqlUserStore) UpdateMfaActive(userId string, active bool) store.StoreCh
 	})
 }
 
+// usersQuery is a starting point for all queries that return one or more Users.
+var usersQuery *querybuilder.Query = querybuilder.
+	Select("u.*").
+	From("Users u")
+
 func (us SqlUserStore) Get(id string) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		if obj, err := us.GetReplica().Get(model.User{}, id); err != nil {
-			result.Err = model.NewAppError("SqlUserStore.Get", "store.sql_user.get.app_error", nil, "user_id="+id+", "+err.Error(), http.StatusInternalServerError)
-		} else if obj == nil {
+		query := usersQuery.Where("Id = :UserId").Bind("UserId", id)
```

Comment:
Switched from `Get` to `SelectOne` to leverage the query, as well as converging on idiomatic error handling.